connector/src/yang/connector/gnmi.py

```diff
@@ -474,11 +454,10 @@ def decode_update(self, update={}, namespace={}):
         if not val:
             val = update.get('val', {}).get('jsonVal', '')
         if not val:
-            log.error('{0} has no values'.format(xpath_str))
+            log.error('JSON has no values')
```

Comment:
`xpath_str` was not declared in this function. so updated the log message without the variable.
@miott please kindly check if this is good.

Reply:
xpath_str is declared at line 410.  It is the return of self.path_elem_to_xpath().

Reply:
@miott7 it's in different function. so copied same code to `decode_update()`.

Reply:
Sorry for the confusion.  ``xpath_str`` is not used or needed in ``decode_update``.

- The ``decode_update`` was separated from ``decode_opfield`` in order to give the user a clear view of a gNMI update return. 
- The ``decode_opfield`` is intended to return a common YANG format of xpath/value to be compared to for expected results.

The error should say "Update has no value".
``log.error('Update has no value')``

Reply:
No problem and thanks for the clarification. updated via 617a02f